src/auth/auth.controller.ts

```diff
@@ -14,7 +14,7 @@ export class AuthController {
     return this.authService.registerUser(user);
   }
 
-  @Roles([UserRole.SuperAdmin])
+  // @Roles([UserRole.SuperAdmin])
```

Comment:
remove comments

Reply:
I have added comment  to disable auth guard for now , for testing purpose , as mentioned in PR description . Not sure if it would still be requires @pnayak97 mam ?